run_all_tests.bat

```diff
@@ -1,3 +1,3 @@
 @echo off
-python tests\run_tests.py
-python tools\cpep8.py

```

Comment:
Did you mean to make these changes? I don't see them mentioned in the commit message, so maybe they were accidental? I'm not a windows user, so I don't know what "py -2" does (aside from the obvious guess). Is that definitely a thing for people with python 2.7 installed?

Reply:
Oh, frack. Yeah, that was just for me. (FYI: I think I actually have a better way now, going thru a VM that has only Python2 installed.)